### PR TITLE
Update some content on the TF page

### PR DESCRIPTION
**Pull request by cnunciato**

### CODE REVIEW

File: themes/default/layouts/migrate/terraform.html [hunk 1]
```diff
@@ -1,43 +1,19 @@
...
             <div class="w-full lg:w-1/2 p-6">
                 <div>
-                    <h2>Increase productivity with programming languages</h2>
-                    <p>Terraform uses a proprietary DSL called HCL to describe and provision infrastructure resources.</p>
+                    <h2>Increase scale and productivity with programming languages</h2>
                     <p>
-                        In Pulumi, you describe infrastructure using familiar languages and tools, providing productivity gains, better abstraction and reuse, and developer
-                        approachability &mdash; while still supporting robust infrastructure as code provisioning across many clouds.
+                        Pulumi and Terraform describe and provision infrastructure differently. Terraform uses a proprietary domain-specific language (DSL) called HashiCorp
+                        Configuration Language (HCL). Pulumi uses general purpose languages that better help you tame cloud complexity, accelerate development velocity, and improve
```
Comment (aaronkao) [inline themes/default/layouts/migrate/terraform.html:13]: Pulumi uses general purpose languages that better help you tame cloud complexity and accelerate development velocity. 

Let's just replace that sentence with this. I think the security claims are too great.

File: themes/default/content/migrate/terraform.md [hunk 2]
```diff
@@ -49,4 +49,134 @@ contact_us_form:
             boosting productivity and taking our business to any cloud environment that our customers need. We
```
Comment (aaronkao) [inline themes/default/content/migrate/terraform.md:49]: what is all of this?

**Review by aaronkao — APPROVED** [review]

File: themes/default/content/migrate/terraform.md (hunk 2 above)
Comment (cnunciato) [reply themes/default/content/migrate/terraform.md:49]: Ah, I'll remove this in the follow-up -- it was data that was used to populate the template. I just overlooked it. (It's no longer used.)

File: themes/default/layouts/migrate/terraform.html (hunk 1 above)
Comment (joeduffy) [reply themes/default/layouts/migrate/terraform.html]: Why are security claims too great?

**Pull request merged**